src/fmnoise/coldbrew.clj

```diff
@@ -52,14 +52,29 @@
    [db date id]
    (query-customer-ltv db date id))
   "
-  [name args cache-key & fn-body]
-  (let [fname (gensym (str name "-cached-"))]
+  [name & fdecl]
```

Comment:
`fdecl`... What is it? 🤔😅  Maybe `body` will be enough? 😅

Reply:
fdecl = function declaration, inspired by `defn` code from clojure.core
![image](https://user-images.githubusercontent.com/4033391/147393163-211392e8-ac3a-4d89-b930-3b918aefd2b1.png)